### PR TITLE
upstream-contrib: Folders view sort options

### PR DESCRIPTION
**Pull request by soncaokim**
- Add sorting options to Folders view
- Drop the "Scan folder" in the app menu. This implementation is redundant with that availble in Folders view, and is provided with inferior UI

### CODE REVIEW

File: app/src/main/java/com/poupa/vinylmusicplayer/sort/FileSortOrder.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import java.io.File;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * @author SC (soncaokim)
+ */
+public class FileSortOrder {
+    //private static final Comparator<File> BY_NAME = (f1, f2) -> StringUtil.compareIgnoreAccent(f1.getName(),f2.getName());
```
Comment (AdrienPoupa) [inline app/src/main/java/com/poupa/vinylmusicplayer/sort/FileSortOrder.java:22]: can this be removed?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -277,10 +277,22 @@
     <string name="title">Title</string>
     <string name="description">Description</string>
     <string name="device_info">Device info</string>
+    <string name="bug_report_use_account">Send using GitHub account</string>
     <string name="bug_report_manual">Submit the report manually</string>
     <string name="username">Username</string>
     <string name="password">Password</string>
+    <string name="bug_report_no_title">Please enter an issue title</string>
```
Comment (AdrienPoupa) [inline app/src/main/res/values/strings.xml:284]: maybe this should have been in the oops PR? either way it's fine

**Review by AdrienPoupa — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (soncaokim) [reply app/src/main/res/values/strings.xml:284]: It should, I've messed up

File: app/src/main/java/com/poupa/vinylmusicplayer/sort/FileSortOrder.java (hunk 1 above)
Comment (soncaokim) [reply app/src/main/java/com/poupa/vinylmusicplayer/sort/FileSortOrder.java]: done

**Pull request merged**